sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -1645,17 +1818,27 @@ private static String readAsString(ResponseBody body) {
                                            @Header("x-ms-encryption-algorithm") EncryptionAlgorithmType encryptionAlgorithm);
 
         @DELETE("{containerName}/{blob}")
-        Call<ResponseBody> delete(@Path("containerName") String containerName,
-                                  @Path("blob") String blobName,
-                                  @Query("snapshot") String snapshot,
-                                  @Query("timeout") Integer timeout,
-                                  @Header("x-ms-lease-id") String leaseId,
-                                  @Header("x-ms-delete-snapshots") DeleteSnapshotsOptionType deleteSnapshots,
-                                  @Header("If-Modified-Since") DateTimeRfc1123 ifModifiedSince,
-                                  @Header("If-Unmodified-Since") DateTimeRfc1123 ifUnmodifiedSince,
-                                  @Header("If-Match") String ifMatch,
-                                  @Header("If-None-Match") String ifNoneMatch,
-                                  @Header("x-ms-version") String version,
-                                  @Header("x-ms-client-request-id") String requestId);
+        Call<ResponseBody> deleteBlob(@Path("containerName") String containerName,
+                                      @Path("blob") String blobName,
+                                      @Query("snapshot") String snapshot,
+                                      @Query("timeout") Integer timeout,
+                                      @Header("x-ms-lease-id") String leaseId,
+                                      @Header("x-ms-delete-snapshots") DeleteSnapshotsOptionType deleteSnapshots,
+                                      @Header("If-Modified-Since") DateTimeRfc1123 ifModifiedSince,
+                                      @Header("If-Unmodified-Since") DateTimeRfc1123 ifUnmodifiedSince,
+                                      @Header("If-Match") String ifMatch,
+                                      @Header("If-None-Match") String ifNoneMatch,
+                                      @Header("x-ms-version") String version,
+                                      @Header("x-ms-client-request-id") String requestId);
+
+        @DELETE("{containerName}")
+        Call<ResponseBody> deleteContainer(@Path("containerName") String containerName,
+                                           @Query("restype") String restype,
```

Comment:
I commented on using camel case for this parameter above. I guess now I understand where it came from. Are there any guidelines about having a different language parameter name (`resType`) than the REST API name (`restype`)?

Reply:
I think if we follow the convention then `resType` is the one, we can do that as long as this rest contract is handwritten, but this will be auto-generated eventually so it is going to reflect how it present in swagger.